react-app/src/components/projects/ProjectNotificationsTable.tsx

```diff
@@ -77,12 +81,62 @@ const ProjectNotificationsTable = (props: ProjectNotificationsTableProps) => {
 
   if (!props.rows) return <></>;
 
+  // Prepare values for Enhanced Referral Notification fields
+  const initalERN = props.rows.find(
+    (row) => row.subject === 'ACTION REQUIRED - Notification of Surplus Real Property',
+  );
+  const thirtyDayERN = props.rows.find(
+    (row) =>
+      row.subject ===
+      'Notification of Surplus Real Property - 30 Day Reminder Notification of Surplus Real Property',
+  );
+  const sixtyDayERN = props.rows.find(
+    (row) =>
+      row.subject ===
+      'Notification of Surplus Real Property - 60 Day Reminder Notification of Surplus Real Property',
+  );
+  const nintyDayERN = props.rows.find(
+    (row) =>
+      row.subject ===
+      'Notification of Surplus Real Property - Completion of 90 Day Enhanced Referral Period for Notification of Surplus Real Property',
+  );
+
```

Comment:
I feel like it would be better if we could look these up in a less brittle way, like based off the template id.

Reply:
Changed to use enums.